set.mm

```diff
@@ -61293,6 +61342,14 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
   frel $p |- ( F : A --> B -> Rel F ) $=
     ( wf wfn wrel ffn fnrel syl ) ABCDCAECFABCGACHI $.
 
+  ${
+    freld.1 $e |- ( ph -> F : A --> B ) $.
+    $( A mapping is a relation.  (Contributed by Glauco Siliprandi,
```

Comment:
```suggestion
    $( A function with domain and codomain is a relation.  (Contributed by Glauco Siliprandi,
```
That's our (somewhat arbitrary) convention to distinguish our various definitions:
- Fun F : a function
- F Fn A : a function with domain
- F : A --> B : a function with domain and codomain

Reply:
I tried to use these specific wordings as much as possible/meaningful in my new theorems. However,  I think it is not always necessary to use the full name, e.g., if the word "codomain" appears already within the comment: "The codomain of a function is .." should be sufficient compared with "The codomain of a function with domain and codomain is .."

Reply:
You're right, I think you described a good balance.